src/components/Button/Button.tsx

```diff
@@ -27,11 +26,15 @@ export const Button = ({
 }: Props): ReactElement => {
   return (
     <button
-      className={cn('btn', className, {
-        'btn--small': size === 'small',
-        'btn--large': size === 'large',
-        'btn--active': active,
-      })}
+      className={cn(
+        'bg-[#FFC600] border-solid border-[#FFC600] rounded-md py-1.5 px-3 text-white border-2 text-[16px] hover:text-[#5A6675] hover:cursor-pointer',
+        className,
+        {
+          'text-[12px] py-1 px-2': size === 'small',
+          'text-[20px] py-2 px-4': size === 'large',
+          'text-[#5A6675] cursor-pointer': active,
+        }
+      )}
       data-testid={dataTestId}
       disabled={disabled}
       id={id}
```

Comment:
not exactly related to your changes but I think we should update this component to inherit the props (`disabled`, `id`, `onClick` e.g.) from the `button` element so that you wouldn't need to manually specify and pass them around again. Should clean up the code a bit 

something like this could work I reckon

```
interface Props extends React.ComponentPropsWithoutRef<'button'> {
  active?: boolean;
  size?: 'small' | 'medium' | 'large';
}

export const Button = ({ children, active = false, size = 'medium', className, ...rest }: Props): ReactElement => {
  return (
    <button
      className={cn(
        'bg-[#FFC600] border-solid border-[#FFC600] rounded-md py-1.5 px-3 text-white border-2 text-[16px] hover:text-[#5A6675] hover:cursor-pointer',
        className,
        {
          'text-[12px] py-1 px-2': size === 'small',
          'text-[20px] py-2 px-4': size === 'large',
          'text-[#5A6675] cursor-pointer': active,
        }
      )}
      {...rest}
    >
      {children}
    </button>
  );
};
```

Reply:
Yeah I agree that cleans the code up nicely. I'll implement it as part of this